pulp_smash/pulp3/constants.py

```diff
@@ -22,13 +22,9 @@
 
 DISTRIBUTION_PATH = urljoin(BASE_PATH, "distributions/")
 
-DOWNLOAD_POLICIES = ("immediate", "on_demand", "streamed")
+DOWNLOAD_POLICIES = ("immediate", )
```

Comment:
This name can be misleading. Since it is plural. I would suggest:
```suggestion
IMMEDIATE_DOWNLOAD_POLICY = ("immediate", )
```

Reply:
I pushed this fix, but what about just making it

`IMMEDIATE_POLICY = "immediate"` since it's single now.